application/views/logbookadvanced/index.php

```diff
@@ -20,9 +33,17 @@
     <?php
     echo "var homegrid ='" . strtoupper($homegrid[0]) . "';";
     if (!isset($options)) {
+		//Set userdata for display settings to pass to JS functions
+		if($this->session->userdata('user_show_log_utc') == 1){
+			$showUTC = "true";
+		}else{
+			$showUTC = "false";
+		}
+		$timezone = $this->session->userdata('user_timezone');
         $options = "{
             \"datetime\":{\"show\":\"true\"},
-            \"de\":{\"show\":\"true\"},
+			\"showutc\":{\"show\":\"$showUTC\"},
+			\"timezone\":{\"value\":\"$timezone\"},            \"de\":{\"show\":\"true\"},
```

Comment:
I don't know what you are doing here exactly but this does not look right to me.